site/_wiki/FAQ/Windows.md

```diff
@@ -129,13 +129,13 @@ It is likely because you are running OpenTabletDriver with the Administrator acc
 
 ### OpenTabletDriver is completely blank {#rtss}
 
-If you are experiencing random crashes with no warning, or the interface is completely blank and/or not accepting input, and you have RivaTuner Statistics Server installed, it may be hooking into OpenTabletDriver and prevents OpenTabletDriver.UX.Wpf from functioning correctly.
+If you are experiencing random crashes with no warning, or the interface is completely blank and/or not accepting input, and you have RivaTuner Statistics Server installed, it may be hooking into OpenTabletDriver and therefore prevents {% otdexe windows-ui %} from functioning correctly.
```

Comment:
Oops, the `therefore` change should have been included in #110. If the change is not liked (or you want clean history) I can rebase this out.

Reply:
It's trivial, keep it.